membership-common/src/main/scala/com/gu/memsub/subsv2/reads/SubPlanReads.scala

```diff
@@ -1,112 +1,40 @@
 package com.gu.memsub.subsv2.reads
-import com.gu.memsub.Product._
-import com.gu.memsub.Product
-import com.gu.memsub.Subscription.ProductId
 import com.gu.memsub.subsv2._
 import com.gu.memsub.subsv2.reads.ChargeListReads.{ProductIds, _}
 import com.gu.memsub.subsv2.reads.CommonReads._
-import com.gu.memsub.{Benefit, BillingPeriod}
-
 import scalaz.Validation.FlatMap._
-import scalaz.std.list._
-import scalaz.syntax.monad._
-import scalaz.syntax.nel._
-import scalaz.syntax.std.boolean._
+import scalaz.ValidationNel
 import scalaz.syntax.std.option._
-import scalaz.{Validation, ValidationNel, \/}
-import scalaz.syntax.apply.ToApplyOps
-
-/** Convert a catalog zuora plan and a subscription zuora plan into some type A Between them, the catalog zuora plan & sub zuora plan have enough info
-  * to construct a Plan
-  */
-trait SubPlanReads[A] {
```

Comment:
since there's only one SubPlanReads now (for the base SubscriptionPlan type) we don't need a trait or implicits, it can be just a normal function.